api/controllers/app/install/controller.go

```diff
@@ -164,10 +180,21 @@ func NewInstallController(opts ...InstallControllerOption) (*InstallController,
 	}
 
 	if controller.appReleaseManager == nil {
-		appReleaseManager, err := appreleasemanager.NewAppReleaseManager(
-			*controller.releaseData.AppConfig,
+		var appReleaseManagerOpts []appreleasemanager.AppReleaseManagerOption
+		appReleaseManagerOpts = append(appReleaseManagerOpts,
 			appreleasemanager.WithLogger(controller.logger),
 			appreleasemanager.WithReleaseData(controller.releaseData),
+			appreleasemanager.WithRuntimeConfig(controller.runtimeConfig),
```

Comment:
app controller should not be aware of runtimeconfig as that's only available in linux targets, not kubernetes targets, and the app controller should be agnostic of that

Reply:
yeah, pretty sure that can be removed now, and its not used anymore. i'll do that real quick.

Reply:
done